.release/ci.hcl

```diff
@@ -0,0 +1,138 @@
+  }
+  github {
+    organization = "hashicorp"
+    repository = "terraform"
+
+    release_branches = [
+      "releng/**" // REMOVEME
+      "main",
+      "release/**",
+      "build-workflow-dev/**",
+    ]
+  }
+}
+
+event "build" {
+  depends = ["merge"]
+  action "build" {
+    organization = "hashicorp"
+    repository = "terraform"
+    workflow = "build"
+  }
+}
+
+// Read more about what the `prepare` workflow does here:
+// https://hashicorp.atlassian.net/wiki/spaces/RELENG/pages/2489712686/Dec+7th+2022+-+Introducing+the+new+Prepare+workflow
+event "prepare" {
+  depends = ["build"]
+
+  action "prepare" {
+    organization = "hashicorp"
+    repository   = "crt-workflows-common"
+    workflow     = "prepare"
+    depends      = ["build"]
+  }
+
+  notification {
+    on = "fail"
+  }
+}
+
+## These are promotion and post-publish events
+## they should be added to the end of the file after the verify event stanza.
+
+event "trigger-staging" {
+// This event is dispatched by the bob trigger-promotion command
+// and is required - do not delete.
+}
+
+event "promote-staging" {
+  action "promote-staging" {
+    organization = "hashicorp"
+    repository = "crt-workflows-common"
+    workflow = "promote-staging"
+    config = "release-metadata.hcl"
+  }
+
+  notification {
+    on = "always"
+  }
+}
+
+event "promote-staging-docker" {
+  depends = ["promote-staging"]
+  action "promote-staging-docker" {
+    organization = "hashicorp"
+    repository = "crt-workflows-common"
+    workflow = "promote-staging-docker"
+  }
+
+  notification {
+    on = "always"
+  }
+}
+
+// event "trigger-production" {
+// // This event is dispatched by the bob trigger-promotion command
+// // and is required - do not delete.
+// }
+
+// event "promote-production" {
+//   depends = ["trigger-production"]
+//   action "promote-production" {
+//     organization = "hashicorp"
+//     repository = "crt-workflows-common"
+//     workflow = "promote-production"
+//   }
+
+//   notification {
+//     on = "always"
+//   }
+// }
+
+// event "promote-production-docker" {
+//   depends = ["promote-production"]
+//   action "promote-production-docker" {
+//     organization = "hashicorp"
+//     repository = "crt-workflows-common"
+//     workflow = "promote-production-docker"
+//   }
+
+//   notification {
+//     on = "always"
+//   }
+// }
+
+// event "promote-production-packaging" {
+//   depends = ["promote-production-docker"]
+//   action "promote-production-packaging" {
+//     organization = "hashicorp"
+//     repository = "crt-workflows-common"
+//     workflow = "promote-production-packaging"
+//   }
+
+//   notification {
+//     on = "always"
+//   }
+// }
+
+// event "update-ironbank" {
+//   depends = ["bump-version-patch"]
+//   action "update-ironbank" {
+//     organization = "hashicorp"
+//     repository = "crt-workflows-common"
+//     workflow = "update-ironbank"
+//   }
+
+//   notification {
+//     on = "always"
+//   }
+// }
+
```

Comment:
after discussing with @terraform, we've decided to opt for option #2. as such, we'll be adding the event call here!

Reply:
I think we'd just as soon have it live in this repo, unless that's a bad idea from a secrets handling perspective! Using `terraform-releases` would also be fine if we need to have it in a private repo for security reasons.

Reply:
Only private repos have access to our internal Vault cluster which is where we'd want the secret to live :)